app/controllers/account/appointments_controller.rb

```diff
@@ -80,7 +80,7 @@ def merge_and_complete_appointment_update
       if simultaneous_appointment
         simultaneous_appointment.merge!(@appointment)
         message = "Your existing appointment scheduled for #{helpers.appointment_date_and_time(simultaneous_appointment)} has been updated."
-        MemberMailer.with(member: @member, appointment: simultaneous_appointment).appointment_confirmation.deliver_later
+        MemberMailer.with(member: @member, appointment: simultaneous_appointment).appointment_updated.deliver_later
       else
         message = "Your appointment was scheduled for #{helpers.appointment_date_and_time(@appointment)}."
         MemberMailer.with(member: @member, appointment: @appointment).appointment_confirmation.deliver_later
```

Comment:
The change above is correct! I think we need one more change to this line, though, to handle the case where someone is updating a single appointment but not by attempting to create a new one at the same time.

I think it's [this test](https://github.com/chicago-tool-library/circulate/pull/1257/files#diff-74dda1fc42a3fd017a9def972cb599f695f55488f20abdd8651e1fc52bc962eeR79-R89) that covers this additional case.

Reply:
When you say change to this line, do you mean adding one more case like:
```
elif !simultaneous_appointment & !create_new
  MemberMailer.with....appointment_updated
```

Reply:
Yeah, that's exactly what I'm thinking. We could add an `updating` (or whatever) parameter to `merge_and_complete_appointment_update` and pass in `true` from the call in `#update` and `false` from the one in `#create`.

In the case where `updating` is false, we'd want to continue to send the `appointment_confirmation` email so that folks get that email when they create an appointment for the first time.